portfolio/src/main/java/com/google/sps/data/CommentUtility.java

```diff
@@ -104,31 +122,29 @@ public static void postComment(String collectionID, HttpServletRequest request)
     commentEntity.setProperty("name", name);
     commentEntity.setProperty("comment", comment);
     commentEntity.setProperty("timestamp", timestamp);
+
+    // Save comment image if it exists
+    String imageUrl = getUploadedFileUrl(request, "image");
+
+    System.out.println(imageUrl);
+    if (imageUrl != null) {
```

Comment:
Can you set null to a property? I have a feeling that this if statement is not necessary, but I am not sure.

Reply:
Yes I can. Good catch.

Reply:
Fixed it.